resources/default.rb

```diff
@@ -2,8 +2,8 @@
 default_action :add
 
 attribute :file, kind_of: String, name_attribute: true
-attribute :user, kind_of: String, required: true
-attribute :password, kind_of: String, required: true
+attribute :user, kind_of: String, required: [:add]
```

Comment:
This is probably going to break this for chef 14 now

Reply:
It worked when I tried it.